OpenOversight/tests/conftest.py

```diff
@@ -300,13 +300,20 @@ def add_mockdata(session):
     department = Department(
         name=DEPARTMENT_NAME,
         short_name="SPD",
+        state=random.sample(US_STATE_ABBREVIATIONS, 1)[0],
```

Comment:
This should be [`random.choice`](https://docs.python.org/3.11/library/random.html#random.choice). Although, I would prefer just picking one state

Reply:
What's the reasoning for just picking one? The way I've understood most testing is that it tends to benefit from random inputs (presuming they are inline with the standards of the column), as you're testing more than just one scenario.

Reply:
Problem with random is that it can lead to flaky tests (sometimes passing, sometimes not) which are much worse to fix than a test that always fails

Reply:
It CAN lead to flakey tests presuming you are not using your random assignments properly or using bad data in your random examples. Regarding states, that won't be the case (it is ALWAYS one of the 50 state abbreviations).  Regarding other values, it's important you choose the correct [provider](https://faker.readthedocs.io/en/master/providers.html). The use of fake data within testing specifically is something used fairly widely and helps us have data within our application that is closer to the real thing.

Should we use an incorrect testing provider, we'll definitely get some errors, but I haven't seen the case thus far.